commands/create_cluster.go

```diff
@@ -419,6 +395,13 @@ func createDefinitionFromFlags(args addClusterArguments) clusterDefinition {
 		def.ReleaseVersion = args.releaseVersion
 	}
 
+	if def.Scaling.Min > 0 && args.workersMin == 0 {
+		args.workersMin = def.Scaling.Min
+	}
+	if def.Scaling.Max > 0 && args.workersMax == 0 {
+		args.workersMax = def.Scaling.Max
+	}
+
```

Comment:
IMO it is a bit confusing to read taking into account the following conditionals. But I don't have a better solution

Reply:
All this code is definitely not nice. Our main goal here is to get the feature out of the door. I want to collectively put more love into our client side code in the future. But this is a general effort we should make together and takes a long time. We are currently developing an idea for the general layout and structure of command line tools and want to apply it then accordingly. Step by step. For now it hurts here and there. So we have to deal with it.